---
layout: default
title: Leduc et al, 2022
nav_order: 5
permalink: Leduc_et_al_2022
description: "Exploring functional protein covariation across single cells using nPOP | Slavov Laboratory"
parent: Single-cell protein data
---
{% include social-media-links.html %}


## Article: [Leduc et al, 2022](https://www.biorxiv.org/content/10.1101/2021.04.24.441211v3)
* Code available at: [github.com/SlavovLab/nPOP](https://github.com/SlavovLab/nPOP)
* **Peer reviewed article:** Leduc A, Huffman RG, Cantlon J, Khan S, Slavov N, [Exploring functional protein covariation across single cells using nPOP](https://genomebiology.biomedcentral.com/articles/10.1186/s13059-022-02817-5) *Genome Biol* **23**, 261 [10.1186/s13059-022-02817-5](https://doi.org/10.1186/s13059-022-02817-5)
 * **Perspective:**  Slavov N., [Learning from natural variation across the proteomes of single cells](https://doi.org/10.1371/journal.pbio.3001512), *PLoS Biology* 20(1): e3001512


### Single-cell proteomics methods: [pSCoPE](pSCoPE) and [plexDIA](plexDIA)
**Sample preparation method: [nPOP](https://scp.slavovlab.net/nPOP)**

### Model systems:  Cell lines of monocytes (U937 cells) and melanoma cells (WM989-A6-G3).


&nbsp;

[Supplementary tables](#supp_tables){: .btn .fs-5 .mb-4 .mb-md-0 .mr-2 }
[RAW Data](#raw_data){: .btn .fs-5 .mb-4 .mb-md-0 .mr-2 }
[Code](https://github.com/SlavovLab/nPOP){: .btn .fs-5 .mb-4 .mb-md-0 .mr-2 }
[Protocol](https://www.protocols.io/view/npop-4r3l24r7qg1y/v1){: .btn .fs-5 .mb-4 .mb-md-0 .mr-2 }
[Processed Data](#proc_data){: .btn .fs-5 .mb-4 .mb-md-0 .mr-2 }
[Videos](#talks){: .btn .fs-5 .mb-4 .mb-md-0 .mr-2 }

<h2 style="letter-spacing: 2px; font-size: 26px;" id="supp_tables" >Supplementary tables</h2>

* [Supp Table 1: Data Organization](https://docs.google.com/spreadsheets/d/1wtPtyAX8wykhplSDhEvv4S6lJEMqTOkT/edit?usp=sharing&ouid=105737144695616543623&rtpof=true&sd=true)
  - A table describing all data from nPOP paper including the MassIVE repository, processed data, and supplemental data
* [Supp Software 2: CellenONE instrument files](https://drive.google.com/drive/folders/14D_wmMCJRE5Zvf7XG-_nwKx4LY8FwQFu?usp=sharing)
  - All CellenONE software files needed for performing nPOP
* [Supp Video 3: Video of Pickup](https://drive.google.com/file/d/1dYDNEFNUy6tT8Muw0C5UB8-QK0u1gfbV/view?usp=sharing)
  - A video of the cellenONE gathering samples off slides and dispensing into a 384 well plate for subsequent analysis
* [Supp Data 4: Meta data](https://docs.google.com/spreadsheets/d/1JtkihjpyDzy3eamB2tLc6-YSjzENClQh/edit?usp=sharing&ouid=105737144695616543623&rtpof=true&sd=true)
  - Meta data for pSCoPE and plexDIA runs
* [Supp Data 5: pSCoPE DO-MS report](https://drive.google.com/file/d/1gU6StqmiP1MNXDzwnYrD-ta5EIpmnKaG/view?usp=sharing)
  - Protein set enrichment analysis between melanoma sub-populations
* [Supp Table 6: CDC analysis](https://docs.google.com/spreadsheets/d/1XFOsxSQF8zIROELmdGp3l1FRCzpq5I7l/edit?usp=sharing&ouid=105737144695616543623&rtpof=true&sd=true)
  - Cell cycle analysis containing differential proteins from bulk, proteins used to create CDC markers, proteins that correlate significantly to CDC phases, and protein set enrichment computed from CDC correlations
* [Supp Table 7:  Melanoma subpopulation analysis](https://docs.google.com/spreadsheets/d/1KAT2Xt0LlfNoB1MUkGRKZcrNWHuXEmyg/edit?usp=sharing&ouid=105737144695616543623&rtpof=true&sd=true)
  - Differentially abundant proteins between melanoma sub-populations and protein set enrichment analysis
* [Supp Figures 8: Supplemental figures](https://drive.google.com/file/d/11Regaf55cxgbDdRzfoIbtuhxlC0T4vj-/view?usp=sharing)
  - The three supplemental figures from the paper
* [Supp Data 9: Processed data](https://drive.google.com/file/d/1OlChCyz6HR_Z5Kc9Z4DrgvE1-DV74IMx/view?usp=sharing)
  - Normalized and imputed protein x single cell matrices for pSCoPE and plexDIA data



   &nbsp;

   &nbsp;


<h2 style="letter-spacing: 2px; font-size: 26px;" id="raw_data" >Raw Data from experiments benchmarking nPOP</h2>

* **MassIVE Repository:**
  - [Description of Repository](https://docs.google.com/spreadsheets/d/1wtPtyAX8wykhplSDhEvv4S6lJEMqTOkT/edit?usp=sharing&ouid=105737144695616543623&rtpof=true&sd=true)
  - [**http:**  MSV000089159](https://massive.ucsd.edu/ProteoSAFe/dataset.jsp?task=158ee6b0cc0b4d8b9f1883b3917793fb)
  - [**ftp:** &nbsp; MSV000089159](ftp://massive.ucsd.edu/MSV000089159)


  &nbsp;

  &nbsp;

<h2 style="letter-spacing: 2px; font-size: 26px;" id="proc_data" >Processed Data from experiments benchmarking nPOP</h2>

* [Processed Data](https://drive.google.com/drive/folders/12-H2a1mfSHZUGf8O50Cr0pPZ4zIDjTac?usp=sharing)
   - This folder contains all processed data necessary for performing the analysis, as well as outputs from analysis

   &nbsp;

   &nbsp;


<h2 style="letter-spacing: 2px; font-size: 26px;" id="talks" >Recorded video presentations</h2>

<iframe width="560" height="315" src="https://www.youtube.com/embed/DJ1U_KpMNcY" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>


&nbsp;

&nbsp;

<iframe width="560" height="315" src="https://www.youtube.com/embed/ped9A3rsJdw" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>

&nbsp;

&nbsp;

&nbsp;

&nbsp;

&nbsp;

&nbsp;

&nbsp;

&nbsp;

&nbsp;

&nbsp;

&nbsp;

&nbsp;

&nbsp;
