---
layout: default
title: 2024, BayesPG
nav_order: 3
permalink: BayesPG
description: "BayesPG is a Bayesian model of transcript and protein abundance that systematically accounts for technical variation and infers biological differences | Slavov Laboratory"
parent: Computational analysis
---
{% include social-media-links.html %}

# QuantQC

## Quantitative evaluation and processing pipeline by [Khan, Elcheikhali et al, 2024][BayesPG_Preprint]
 * Code available at: [github.com/SlavovLab/BayesPG](https://github.com/SlavovLab/BayesPG)
 * Khan S & Elcheikhali M, Leduc A, Huffman R, Derks J, Franks A & Slavov N, [Inferring post-transcriptional regulation within and across cell types in human testis](https://www.biorxiv.org/content/...)
<!--
* **Peer-reviewed article:** Leduc A, Luke Khoury, Joshua Cantlon, Slavov N. (2024)
Massively parallel sample preparation for multiplexed single-cell proteomics using nPOP, *Nature Protocols*, doi: [10.1038/s41596-024-01033-8](https://doi.org/10.1038/s41596-024-01033-8), [blog](https://blog.slavovlab.net/2024/08/09/flexibility-meets-quality-and-scale/)

-->

&nbsp;




&nbsp;

QuantQC is a Bayesian software for modeling single-cell RNA and protein abundance that systematically accounts for technical variation and infers biological differences.  

---



&nbsp;

## Implementation
BayesPG is available from [github.com/SlavovLab/BayesPG](https://github.com/SlavovLab/BayesPG). Try it out and give us feedback!

&nbsp;  

&nbsp;


&nbsp;  

&nbsp;

&nbsp;


&nbsp;

&nbsp;

[BayesPG_Preprint]: https://www.biorxiv.org/content/10.1101/2023.11.27.568927v1 "Bayesian software for modeling single-cell RNA and protein abundance that systematically accounts for technical variation and infers biological differences"


&nbsp;

&nbsp;

&nbsp;

&nbsp;

&nbsp;

&nbsp;

&nbsp;

&nbsp;

&nbsp;

&nbsp;

&nbsp;
